.github/workflows/ci.yml

```diff
@@ -40,14 +41,18 @@ jobs:
         with:
           compiler: ${{ matrix.dc }}
 
-      - name: Setup C++
+      - name: 'Non-Linux: Setup C++'
```

Comment:
I don't understand why this change is needed, given that it was working fine before. Is there something special about Ubuntu 22.04?

Reply:
If you can explain the problem, I can fix it in setup-cpp.

Reply:
The point is to test whether it's working out of the box on at least 2 different Ubuntu versions, by installing the `libclang-dev` apt package, instead of doing some special GitHub-Actions setup that noone in real life uses on his dev box. (And had to work around by creating `libclang.so` symlinks manually, as we did at Symmetry.)